src/main/riddl/Members/members.riddl

```diff
@@ -127,6 +127,60 @@ context MembersContext  is {
             }
         }
 
+        state Draft is {
+          fields {
+            info: MembersContext.Info,
+            meta: MembersContext.MetaInfo
+          }
+
+          handler DraftMemberHandler {
+            on command RegisterMember {then error "Member has already been registered."}
+            on command UpdateMemberStatus {
+              example DraftMemberToActiveMember is {
+                given "entity is transitioning from Draft to Active and required fields are non-empty"
+                then "set entity Status to Status field from message"
+                and "set entity lastModifiedBy to actingMember from message"
+                and "set entity lastModifiedOn to today()"
+                and "yield event MemberStatusUpdated"
+              }
+              example DraftMemberToInactiveMember is {
+                given "entity is transitioning from Draft to Inactive and required fields are non-empty"
+                then "set entity Status to Status field from message"
+                and "set entity lastModifiedBy to actingMember from message"
+                and "set entity lastModifiedOn to today()"
+                and "yield event MemberStatusUpdated"
+              }
+              example DraftMemberToSuspendedMember is {
+                given "entity is transitioning from Draft to Suspended and required fields are non-empty"
+                then "set entity Status to Status field from message"
+                and "set entity lastModifiedBy to actingMember from message"
+                and "set entity lastModifiedOn to today()"
+                and "yield event MemberStatusUpdated"
+              }
+              example DraftMemberToTerminatedMember is {
+                given "entity is transitioning from Draft to Terminated and required fields are non-empty"
+                then "set entity Status to Status field from message"
+                and "set entity lastModifiedBy to actingMember from message"
+                and "set entity lastModifiedOn to today()"
+                and "yield event MemberStatusUpdated"
+              }
+              example DraftMemberIllegalStateTransition is {
+                given "entity is transitioning from Draft to Draft"
+                then error "Draft members may not transition to desired state."
+              }
+            }
+            on command UpdateMemberInfo {
+              then "set entity Info to non-Empty fields in UpdateInfo from message"
+              and "set entity lastModifiedBy to actingMember from message"
+              and "set entity lastModifiedOn to today()"
+              and "yield event MemberInfoUpdated"
+            }
+            on query GetMemberData {
+              then "yield Info for entity"
```

Comment:
eventually we will need to worry about permissions here. I think only admins and self should be able to make this call. Or, maybe other roles can get a limited set of info (in which case we would need another aggregate?). For the KalixStudy the way you have it is fine. Just notes for future reference.